rollup/core/classes/RollupFlowBulkProcessor.cls

```diff
@@ -44,18 +44,16 @@ global without sharing class RollupFlowBulkProcessor {
     global Boolean isFullRecordSet;
     @InvocableVariable(label='Is Rollup Started From Parent' description='If the calc records are the parent records, set this to true')
     global Boolean isRollupStartedFromParent;
-    @InvocableVariable(label='Order By (First/Last)' description='First/Last order by field')
+    // TODO - introduce Apex-defined type for Flow to configure FIRST/LAST properties
```

Comment:
Is this something you're still planning to do down the road? Or is this an old `TODO`?

Reply:
Hmm I thought I had reverted that comment since I ended up reusing that field! Let me ensure it’s been removed

Reply:
Ah I see, I had two TODOs - one for the base invocable, one for the bulk processor. Removed!